### ISSUE TITLE
ucf-101 fine-tune from kinetics-400 pretrained model

### ISSUE THREAD

**Issue by taeoh-kim** [MCG-NJU/VideoMAE#71]
Hello.

How do I reproduce K400 (pretrain, 1600ep) -> UCF101 (finetune) performance as shown in the paper (96.1 Top-1 in UCF)?
Should I use the same [config](https://github.com/MCG-NJU/VideoMAE/blob/main/scripts/ucf101/videomae_vit_base_patch16_224_tubemasking_ratio_0.75_epoch_3200/finetune.sh) as UCF101 (pretrain) -> UCF101 (finetune)?

**Comment by YTHmamba** [comment]
Hello, how did you reproduce it？

**Comment by YTHmamba** [comment]
hello, @yztongzhan 。Thank you for your contribution. Could you please tell me  the configuration of finetune UCF101 with accuracy of 96.1 under K400 (pretrain, 1600ep)
